Facebook.iOS/source/Facebook/ApiDefinition.cs

```diff
@@ -326,6 +323,9 @@ interface ApplicationDelegate
 		[Export ("application:openURL:options:")]
 		bool OpenUrl (UIApplication application, NSUrl url, NSDictionary<NSString, NSObject> options);
```

Comment:
I would definitely switch the implementation here where this one becomes the `[Export]`'ed method

```csharp
bool OpenUrl (UIApplication application, NSUrl url, NSDictionary options);
```

and this one the `[Wrap]`'ed one

```csharp
bool OpenUrl (UIApplication application, NSUrl url, NSDictionary<NSString, NSObject> options);
```

Reply:
If you are allowed to do breaking changes just remove/let Thanos make dust with it/kill this one with 🔥 

```csharp
bool OpenUrl (UIApplication application, NSUrl url, NSDictionary<NSString, NSObject> options);
```

It does not really help on anything.

Reply:
I'll switch it and add a "Will be removed on next versions" message to avoid people yelling to Thanos by the moment.